systest_utils/scenarios_manager.py

```diff
@@ -36,117 +45,90 @@ def apply_scenario(self):
         time.sleep(5)
 
     
-
     def apply_fix(self, fix_type):
         """
         apply_fix apply the fix to the cluster
         """
         fix_command= os.path.join(self.scenario_path, self.test_scenario, 'fix_' + fix_type)
         TestUtil.run_command(command_args=fix_command, display_stdout=True, timeout=300)
         time.sleep(5)
+  
 
+    def trigger_scan(self, trigger_by) -> None:
+        """trigger_scan create a new scan action from the backend
 
-    
-    def verify_scenario(self, verify_attack_chains=True, verify_security_risks=False):
+        :param trigger_by: the kind of event that trigger the scan ("cronjob", "scan_on_start")
         """
-        verify_scenario validate the scenario results on the backend
+        Logger.logger.info("triggering a new scan")
+        if trigger_by == "cronjob":
+            self.backend.create_kubescape_job_request(
+                cluster_name=self.cluster,
+                trigger_by=trigger_by,
+                framework_list=["security"],
+                with_host_sensor="true"
```

Comment:
```suggestion
                with_host_sensor="false"
```
security fw doesn't need host scanner